src/comp/pages/CreateProjectPage.tsx

```diff
@@ -1,45 +1,83 @@
 import * as React from "react";
 import { RouteComponentProps } from "react-router-dom";
-import * as styles from "css/comp/pages/CreateProjectPage.module.css";
 import * as CommonStyles from "css/comp/Common.module.css";
 import CreateSendButton from "../SendButton";
-import { TextField } from "@material-ui/core";
+import {
+  Divider,
+  TextField,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemSecondaryAction,
+  IconButton,
+} from "@material-ui/core";
+import { ScrollableTagList } from "../ScrollableTagList";
+import CloseIcon from "@material-ui/icons/Close";
+
+type Site = {
+  title: string;
+  url: string;
+};
 
 type CreateProjectPageState = {
   name: string;
   introduction: string;
+  addingTagName: string;
+  siteNameText: string;
+  siteURLText: string;
+  tagIDs: string[];
+  sites: Site[];
   isNameError: boolean;
   isIntroductionError: boolean;
+  isSiteNameError: boolean;
+  tagIDErrorCode: number;
+  siteURLErrorCode: number;
```

Comment:
```suggestion
  errors: ProjectError[],
```

以下のような型定義を作ってもっとわかりやすくしてほしいです

```
type ProjectError =
  | "name"
  | "introduction"
  | "siteName"
  | TagIDError
  | SiteURLError;
type TagIDErrpr = "blank" | "duplicate" | "unavailableChar";
type SiteURLError = "blank" | "invalid";